core/internal/cltest/cltest.go

```diff
@@ -99,20 +122,36 @@ func NewConfig(t testing.TB) (*TestConfig, func()) {
 	return NewConfigWithWSServer(t, wsserver), cleanup
 }
 
+func NewRandomInt64() int64 {
+	id := rand.Int63()
+	return id
+}
+
 // NewTestConfig returns a test configuration
-func NewTestConfig(t testing.TB) *TestConfig {
+func NewTestConfig(t testing.TB, options ...interface{}) *TestConfig {
 	t.Helper()
 
 	count := atomic.AddUint64(&storeCounter, 1)
 	rootdir := filepath.Join(RootDir, fmt.Sprintf("%d-%d", time.Now().UnixNano(), count))
 	rawConfig := orm.NewConfig()
+
+	rawConfig.Dialect = orm.DialectTransactionWrappedPostgres
+	for _, opt := range options {
+		switch v := opt.(type) {
+		case orm.DialectName:
+			rawConfig.Dialect = v
+		}
+	}
+
+	uniqueRandomID := NewRandomInt64()
+	// Unique advisory lock is required otherwise all tests will block each other
+	rawConfig.AdvisoryLockID = uniqueRandomID
+
 	rawConfig.Set("BRIDGE_RESPONSE_URL", "http://localhost:6688")
 	rawConfig.Set("ETH_CHAIN_ID", 3)
 	rawConfig.Set("CHAINLINK_DEV", true)
 	rawConfig.Set("ETH_GAS_BUMP_THRESHOLD", 3)
-	rawConfig.Set("LOG_LEVEL", orm.LogLevel{Level: zapcore.DebugLevel})
-	rawConfig.Set("LOG_SQL", false)
-	rawConfig.Set("LOG_SQL_MIGRATIONS", false)
```

Comment:
Is setting these log levels redundant, or are you turning on more logging? There were complaints before that the test output was too noisy with these turned on. Having lived with them off by default now, I tend to prefer it too.

Reply:
I removed this because I needed SQL and debug logging turned on in order to debug the tests.

I left it out because it seems to me that it would be nice to allow people to set their own logging levels.

With these lines gone, tests will inherit the logging levels from the enviroment. If you want less logging you can set `LOG_LEVEL=fatal` and if you want more you can set `LOG_LEVEL=debug`.